### PR TITLE
Translate on different connection

### PR DESCRIPTION
**Pull request by nsaumini**
This is a fix for following [issue](https://github.com/dimsav/laravel-translatable/issues/283). The solution is derived from one of the user's work around.

### CODE REVIEW

File: tests/TranslatableTest.php [hunk 1]
```diff
@@ -634,4 +634,20 @@ public function test_it_uses_fallback_locale_if_default_is_empty()
...
+        $country->{'name:sg'} = 'Singapore';
+        $this->assertTrue($country->save());
+
+        $countryId = $country->id;
+
+        $country = new Country();
+        $country->setConnection('mysql2');
+        $sgCountry = $country->find($countryId);
+        $this->assertEquals($sgCountry->translate('sg')->name, 'Singapore');
+    }
```
Comment (dimsav) [inline tests/TranslatableTest.php]: Nice! Also let's assert here that by using the default connection, the Singapore record cannot be found in the database.

Comment (nsaumini) [inline tests/TranslatableTest.php]: @dimsav Noted & added more assertions as you requested and also I added a empty check before `setConnection`

**Comment by dimsav** [comment]
Awesome, thanks! We still have to update the changelog and the docs for the release.

**Comment by nsaumini** [comment]
@dimsav Thanks. I'm not sure how/ where to update the changelog & docs for the release. Can you guide me?

**Comment by dimsav** [comment]
Sure! Here's the [changelog](https://github.com/dimsav/laravel-translatable/blob/master/changelog.md) (we should make a 7.3 release) and the [docs](https://github.com/dimsav/laravel-translatable/blob/master/readme.md#features-list) (we should add somewhere the new method.

**Comment by dimsav** [comment]
Released :-)

**Comment by nsaumini** [comment]
Thanks for the support :)

**Pull request merged**